docs/examples/iq_plot_listen.py

```diff
@@ -0,0 +1,259 @@
+# Copyright 2020 SuperDARN Canada, University of Saskatchewan
+# Author: Kevin Krieger & Ashton Reimer
+"""
+Utilities for plotting and listening to DMAP IQ files
+
+Exceptions
+----------
+
+Functions
+---------
+plot_iq
+
+iq_to_wav
+
+References: https://www.garykessler.net/library/file_sigs.html
+"""
+import pydarn
+import binascii
+import numpy as np
+import datetime as dt
+import soundfile
+import matplotlib.pyplot as plt
+import argparse as ap
+import os
+
+
+def iq_to_wav(records, save_directory, sequences_to_play=None):
+    """
+    Create one wav file for all records.
+
+    This should plot the real and imag (I and Q) data in the left
+    and right channels so you can listen in stereo (not yet implemented)!
+    The sampling rate is currently hard-coded at 3333Hz,
+    this should be fixed to play at the actual sampling rate, for most files it will be correct.
+    :param records: pydarn iqdat records, as returned from read_iqdat()
+    :param save_directory: Filepath to save the wav files to
+    :param sequences_to_play: up to how many sequences should be written to wav file for each record?
+    :return:
+    """
+    first = records[0]
+    stid = first['stid']
+    radar_abbrev = pydarn.SuperDARNRadars.radars[stid].hardware_info.abbrev
+    record_datetime = dt.datetime.strptime(first['origin.time'], "%a %b %d %H:%M:%S %Y")
+    record_datetime = record_datetime.strftime("%Y%m%d.%H%M.%S")
+    wav_filename = save_directory + '/' + record_datetime + '.' + radar_abbrev + \
+                   '.bm' + str(first['bmnum']) + '.wav'
+
+    # Signals will contain the actual wav file samples
+    signals = []
+    for record in records:
+        if sequences_to_play is None:
+            # All of them!
+            sequences_to_play = int(record['seqnum'])
+        if not isinstance(sequences_to_play, int):
+            print("Need to specify an integer for sequences_to_play, plotting all.")
+            sequences_to_play = record['seqnum']
+        sequences_to_play = max(sequences_to_play, record['seqnum'])
+
+        data = np.copy(record['data'])
+        data = data.reshape((int(record['seqnum']), int(record['chnnum']), int(record['smpnum']), 2))
+        for sequence in range(sequences_to_play):
+            iq_real = np.array([x[0] for x in data[sequence][0]])
+            iq_imag = np.array([x[1] for x in data[sequence][0]])
```

Comment:
```suggestion
```
unused variable

Reply:
left this in for future stereo option which I have now implemented :)